tfhe/src/integer/u256.rs

```diff
@@ -672,6 +733,41 @@ mod tests {
         assert_eq!(res, u128_max);
     }
 
+    #[test]
+    fn test_div_rem() {
+        let u64_max = U256::from(u64::MAX);
+        let (expected_q, expected_r) = (u64::MAX / u64::MAX, u64::MAX % u64::MAX);
+        assert_eq!(u64_max / u64_max, U256::from(expected_q));
+        assert_eq!(u64_max % u64_max, U256::from(expected_r));
+
+        let mut rng = rand::thread_rng();
+        for _ in 0..5 {
+            let a = rng.gen::<u128>();
+            let b = rng.gen::<u128>();
+
+            let res_q = U256::from(a) / U256::from(b);
+            let res_r = U256::from(a) % U256::from(b);
+            let expected_q = a / b;
+            let expected_r = a % b;
+            assert_eq!(res_q, U256::from(expected_q));
+            assert_eq!(res_r, U256::from(expected_r));
+        }
+
+        let u128_max = U256::from(u128::MAX);
+        let res_q = u128_max / U256::from(3284723894u64);
+        let res_r = u128_max % U256::from(3284723894u64);
+        let expected_q = U256::from(103595424730374145554705368314u128);
+        let expected_r = U256::from(701916739u128);
+        assert_eq!(res_q, expected_q);
+        assert_eq!(res_r, expected_r);
+
+        let u256_max = U256::MAX;
+        let res_q = u256_max / U256::ONE;
+        let res_r = u256_max % U256::ONE;
+        assert_eq!(res_q, u256_max);
+        assert_eq!(res_r, U256::ZERO);
+    }
+
```

Comment:
can we add 256 bits examples taken from python ?

Reply:
I added 2